test/server/test-error-test.js

```diff
@@ -0,0 +1,72 @@
+var expect             = require('chai').expect;
+var buildTestError     = require('../../lib/test-error');
+var plainTextDecorator = require('../../lib/test-error/plain-text-decorator');
+var readFile           = require('../../lib/utils/read-file-relative');
+var TYPE               = require('../../lib/test-error/type');
+
+var MAX_STRING_LENGTH = 10;
+
+function compareMsg (expectedMsg, err) {
+    var msg = buildTestError(err, plainTextDecorator, MAX_STRING_LENGTH);
+
+    expect(expectedMsg).to.be.equal(msg);
```

Comment:
Use `expect().eql()` instead. It's consistent with our code base and it's checks for the deep equality.